src/v/archival/archiver_operations_impl.cc

```diff
@@ -61,12 +88,108 @@ class archiver_operations_impl : public archiver_operations_api {
       , _bucket(std::move(bucket))
       , _sg(sg) {}
 
-    /// Return upload candidate(s) if data is available or not_enough_data error
-    /// if there is not enough data to start an upload.
+    /// Return upload candidate(s) if data is available or not_enough_data
+    /// error if there is not enough data to start an upload.
     ss::future<result<reconciled_upload_candidates_list>>
     find_upload_candidates(
```

Comment:
nit: can we please move these outside the class declaration? It will make this more readable with one fewer scope. Same with every other method.